C/services/north/data_load.cpp

```diff
@@ -652,6 +646,7 @@ void DataLoad::configChange(const string& category, const string& newConfig)
 		if (config.itemExists("source"))
 		{
 			setDataSource(config.getValue("source"));
+			m_lastFetched = 0; // Reset last fetch counter on source change
```

Comment:
Isn't this still going to reset the last fetched id when we have a configuration change? Surely we only wan to do this when the actual value of the source item changes?

Reply:
Yes you are correct, I have updated code to reset last fetched id only if source changes.